enterprise/frontend/src/embedding-sdk/store/auth/auth.ts

```diff
@@ -101,37 +102,17 @@ export const refreshTokenAsync = createAsyncThunk(
     url: MetabaseAuthConfig["metabaseInstanceUrl"],
     { getState },
   ): Promise<MetabaseEmbeddingSessionToken | null> => {
-    const customGetRefreshToken =
-      getFetchRefreshTokenFn(getState() as SdkStoreState) ?? null;
+    const state = getState() as SdkStoreState;
+
+    if (getIsSdkIframeEmbedAuth(state)) {
+      return requestSessionTokenFromEmbedJs();
+    }
```

Comment:
We cannot authenticate inside the iframe. This makes it so that we asks for a session token from the parent instead, and skips the sdk's token logic entirely.

See [this thread](https://metaboat.slack.com/archives/C063Q3F1HPF/p1748597244728309) on an alternative API (`privateFetchSessionTokenFn`) that would allow us to not call `requestSessionTokenFromEmbedJs` directly in the SDK, but this adds one more layer of indirection that might be confusing.